main/python2/APKBUILD

```diff
@@ -57,13 +56,14 @@ build() {
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
-		--enable-shared \
-		--with-threads \
 		--enable-ipv6 \
-		--with-system-ffi \
+		--enable-optimizations \
+		--enable-shared \
+		--enable-unicode=ucs4 \
 		--with-system-expat \
+		--with-system-ffi \
 		--with-system-zlib \
-		--enable-unicode=ucs4
+		--with-threads
```

Comment:
Why the move of options here? It makes the diff more confusing than it has to be.

Reply:
I sorted them just like in `main/python3`.